stl/inc/xutility

```diff
@@ -2353,7 +2353,7 @@ namespace ranges {
 
     template <class _Ty, template <class...> class _Template>
     concept _Derived_from_specialization_of = is_object_v<_Ty> && requires(const _Ty& _Obj) {
-        _RANGES _Derived_from_specialization_impl<_Template>(_Obj);
+        _RANGES _Derived_from_specialization_impl<_Template>(_Obj); // qualified to avoid ADL, handle incomplete types

```

Comment:
Not worth resetting testing, but "incompletable types" would be more precise.

Reply:
I was able to cancel the x64 test run after 2 minutes so not too much time was burned.